source/reference/limitations.txt

```diff
@@ -104,6 +107,10 @@ Sharded Clusters
 - ``mongosync`` only syncs indexes that exist on all shards.
 - ``mongosync`` only syncs indexes that have consistent index
   specifications on all shards.
+- You must stop the balancer on a sharded source cluster until mongosync finishes the collection copy phase.
```

Comment:
This should have been in the downstream changes doc, but it's also important that the balancer is off for 15 minutes before mongosync begins, because this is how long chunk migrations might take. Can we add some information on that?

Reply:
Fixed